source/crud/crud.rst

```diff
@@ -589,13 +589,16 @@ Basic
     /**
      * Sends a batch of writes to the server at the same time.
      *
+     * An error MUST be raised if the requests parameter is empty.
```

Comment:
As @ShaneHarvey mentioned in [SPEC-843](https://jira.mongodb.org/browse/SPEC-843?focusedCommentId=1640082&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-1640082), this was already [prohibited in the bulk write spec](https://githb.com/mongodb/specifications/blob/master/source/driver-bulk-update.rst#empty-batch). I don't think this should pose a BC break for any drivers, but I'm willing to change this to SHOULD if necessary.